Analysis/config/2017/sources_tauPOG.cfg

```diff
@@ -11,8 +12,9 @@ hist_cfg: hh-bbtautau/Analysis/config/2017/histograms.cfg
 plot_cfg: hh-bbtautau/Analysis/config/2017/plot_setup.cfg
 plot_page_opt: internal_plot
 jet_ordering: DeepCSV
-tauIdDiscriminator: byIsolationMVArun2017v2DBoldDMwLT2017
-#tauIdDiscriminator: byDeepTau2017v1VSjet
+#tauIdDiscriminator: byIsolationMVArun2017v2DBoldDMwLT2017
+tauIdDiscriminator: byDeepTau2017v1VSjet
```

Comment:
the choice of the discriminator should be consistent with the selected SignalMone. Therefore, please, remove tauIdDiscriminator from the config and take it from signalMode, where needed.